crates/fuel-core/src/database.rs

```diff
@@ -184,11 +189,25 @@ where
     }
 
     #[cfg(feature = "rocksdb")]
-    pub fn open_rocksdb(path: &Path, capacity: impl Into<Option<usize>>) -> Result<Self> {
+    pub fn open_rocksdb(
+        path: &Path,
+        capacity: impl Into<Option<usize>>,
+        state_rewind_policy: StateRewindPolicy,
+    ) -> Result<Self> {
         use anyhow::Context;
-        let db = RocksDb::<Description>::default_open(path, capacity.into()).map_err(Into::<anyhow::Error>::into).with_context(|| format!("Failed to open rocksdb, you may need to wipe a pre-existing incompatible db e.g. `rm -rf {path:?}`"))?;
+        let db = RocksDb::<Description>::default_open(path, capacity.into())
+            .map_err(Into::<anyhow::Error>::into)
+            .with_context(|| {
+                format!(
+                    "Failed to open rocksdb, you may need to wipe a \
+                pre-existing incompatible db e.g. `rm -rf {path:?}`"
+                )
```

Comment:
nit: Do will still want to suggest wiping the DB?

Reply:
I will remove it in a separate PR related to running Database